trtools/mergeSTR/tests/test_mergeSTR.py

```diff
@@ -46,6 +47,20 @@ def __init__(self, chrom, pos, ref, alts=None, info=None):
         self.info = info if info is not None else {}
         self.vcfrecord = DummyRecord(chrom, pos, ref, self.alt_alleles, self.info)
 
+# Test file with list of VCFs
+def test_FileList(args, mrgvcfdir, tmpdir):
+    fname1 = os.path.join(mrgvcfdir, "test_file_gangstr1.vcf.gz")
+    fname2 = os.path.join(mrgvcfdir, "test_file_gangstr2.vcf.gz")
+    args.vcftype = "gangstr"
+    listfile = str(tmpdir / "test.list")
+    f = open(listfile, "w")
+    f.write(fname1+"\n")
+    f.write(fname2+"\n")
+    f.close()
+    args.vcfs_list = listfile
+    args.vcfs = None
+    assert main(args)==0
```

Comment:
in addition to checking that the exit code is non-zero, it might also be good to check that the output appears like you would expect via `assert_same_vcf()`?

```suggestion
    fname1 = os.path.join(mrgvcfdir, "test_file_hipstr1.vcf.gz")
    fname2 = os.path.join(mrgvcfdir, "test_file_hipstr2.vcf.gz")
    args.vcftype = "hipstr"
    listfile = str(tmpdir / "test.list")
    f = open(listfile, "w")
    f.write(fname1+"\n")
    f.write(fname2+"\n")
    f.close()
    args.vcfs_list = listfile
    args.vcfs = None
    assert main(args) == 0
    assert_same_vcf(args.out + '.vcf', mrgvcfdir + "/hipstr_merged.vcf")
```

Reply:
thanks, this is a great suggestion! I added a check like this to the test